channels/topic-career-advice/jr_sr_mid

```diff
@@ -11,9 +11,9 @@ More importantly, provide career benchmarks and a framework for goal setting and
 This is a very young industry that moves very fast, so objective measurements dont exist.
 ## Content
 
-I hear Dan Hannigan can tell you if your a Senior or not
+I hear Dan Hannigan can tell you if you're a Senior or not
```

Comment:
What?

Reply:
If it's not true I guess we can remove it, @danhannigan .

Reply:
Looks like previously set standards are being followed.